cmd/merge.go

```diff
@@ -0,0 +1,76 @@
+/*
+ Copyright 2020 Qiniu Cloud (qiniu.com)
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package cmd
+
+import (
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/tools/cover"
+	"k8s.io/test-infra/gopherage/pkg/cov"
+	"k8s.io/test-infra/gopherage/pkg/util"
+)
+
+var mergeCmd = &cobra.Command{
+	Use:   "merge [files...]",
+	Short: "Merge multiple coherent Go coverage files into a single file.",
+	Long: `merge will merge multiple Go coverage files into a single coverage file.
+merge requires that the files are 'coherent', meaning that if they both contain references to the
+same paths, then the contents of those source files were identical for the binary that generated
```

Comment:
`were` => `are`

Reply:
binary is generated some times ago, so it should be were.

actually i copy the 'long' content from https://github.com/kubernetes/test-infra/blob/master/gopherage/cmd/merge/merge.go. >_<